types/src/function.rs

```diff
@@ -38,13 +40,15 @@ impl FunctionInput {
 #[derive(Debug, Deserialize, Serialize)]
 pub struct FunctionOutput {
     pub name: String,
+    pub option: bool,
     pub description: String,
 }
 
 impl FunctionOutput {
-    pub fn new(name: impl Into<String>, description: impl Into<String>) -> Self {
+    pub fn new(name: impl Into<String>, option: bool, description: impl Into<String>) -> Self {
```

Comment:
Do we need to think about future compatibility? For example, we may have some "decrypted" input file, and we can have an argument called `property`, and `property='do'` stands for decrypted, optional.